templates/base.html

```diff
@@ -84,6 +84,10 @@
               </li>
             {% endif %}
 
+            {% if is_participating_this_year %}
+              <li><a href="{% url 'cloud_access' %}">Chmura</a></li>
```

Comment:
This should be available only if owncloud is actually configured, but we can fix this later too